.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,7 +7,7 @@ behind them.
 
   - [ ] I've read [CONTRIBUTING.md](https://github.com/xmonad/xmonad/blob/master/CONTRIBUTING.md)
 
-  - [ ] I tested my changes with [xmonad-testing](https://github.com/xmonad/xmonad-testing)
+  - [ ] I tested my changes with commit XXX
```

Comment:
Maybe also add something like:
`I tested my changes with the latest git version`

Is that appropriate?

Reply:
> `I tested my changes with the latest git version`

I was thinking about this, but what is "the latest git version" may very well change in the course of the pr, especially when it's a longer affair, so I think explicitly specifying the commit is better. 

> Also specify `xmonad` and `xmobar` version.
> For example, some users will mix `xmonad-contrib git` with `xmonad 0.15`.

That's a good suggestion, thanks!  I don't think xmobar is necessary, but the version of the xmonad core would certainly be interesting.

Reply:
I think mentioning the commit is better in this case, but maybe also mention a version? Something like

  - I tested my changes
    - [ ] With `xmonad`: version XX  (commit XXX, if building from git)
    - [ ] With `xmonad-contrib`: version XX (commit XXX, if building from git)


that might be too verbose though

Reply:
I don't think verbosity is really something that we should care about in a template like this; as long as we don't paste entire essays there :)

I'm curious how common it is to be on a non-git xmonad version when submitting pr's.  If that's a thing that people generally do then it would definitely be worth adding (not for contrib though, I'd expect people to have tested their pr's against _some_ version of contrib git)